<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="20434" started-at="2021-07-21T01:17:44 IST" finished-at="2021-07-21T01:18:05 IST">
    <groups>
    </groups>
    <test name="Default test" duration-ms="20434" started-at="2021-07-21T01:17:44 IST" finished-at="2021-07-21T01:18:05 IST">
      <class name="com.amdtelecom.route.Testdata.SendSMSTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.amdtelecom.route.Testdata.SendSMSTest@5f9d02cb]" name="setUp" is-config="true" duration-ms="15076" started-at="2021-07-21T01:17:44 IST" finished-at="2021-07-21T01:17:59 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="SendSMSTest1()[pri:0, instance:com.amdtelecom.route.Testdata.SendSMSTest@5f9d02cb]" name="SendSMSTest1" duration-ms="4438" started-at="2021-07-21T01:17:59 IST" description="sending SMS " finished-at="2021-07-21T01:18:04 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.amdtelecom.route.Testdata.SendSMSTest.SendSMSTest1(SendSMSTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SendSMSTest1 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.amdtelecom.route.Testdata.SendSMSTest@5f9d02cb]" name="tearDown" is-config="true" duration-ms="849" started-at="2021-07-21T01:18:04 IST" finished-at="2021-07-21T01:18:05 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.amdtelecom.route.Testdata.SendSMSTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
